DependencyInjection/DoctrineExtension.php

```diff
@@ -800,7 +800,7 @@ protected function loadOrmCacheDrivers(array $entityManager, ContainerBuilder $c
     private function loadPropertyInfoExtractor($entityManagerName, ContainerBuilder $container)
     {
         $propertyExtractorDefinition = $container->register(sprintf('doctrine.orm.%s_entity_manager.property_info_extractor', $entityManagerName), DoctrineExtractor::class);
-        if (interface_exists(PropertyInitializableExtractorInterface::class)) {
+        if ((new ReflectionClass(DoctrineExtractor::class))->hasProperty('entityManager')) {
```

Comment:
What about using `property_exists()` here?

Reply:
Even better! No need to bother with reflection! 👍